\documentclass[12pt,letterpaper]{article}

%Packages
\usepackage{natbib}
\usepackage{pdflscape}
\usepackage{fixltx2e}
\usepackage{textcomp}
\usepackage{fullpage}
\usepackage{float}
\usepackage{latexsym}
\usepackage{url}
\usepackage{epsfig}
\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{amsmath}
\usepackage{bm}
\usepackage{array}
\usepackage[version=3]{mhchem}
\usepackage{ifthen}
\usepackage{caption}
\usepackage{hyperref}
\usepackage{amsthm}
\usepackage{amstext}
\usepackage{enumerate}
\usepackage[osf]{mathpazo}
\usepackage{dcolumn}
\usepackage{lineno}
\usepackage{color}
\usepackage[usenames,dvipsnames]{xcolor}
\pagenumbering{arabic}

%Pagination style and stuff
%\linespread{2} 

\raggedright
\setlength{\parindent}{0.5in}
\setcounter{secnumdepth}{0} 
\renewcommand{\section}[1]{%
\bigskip
\begin{center}
\begin{Large}
\normalfont\scshape #1
\medskip
\end{Large}
\end{center}}
\renewcommand{\subsection}[1]{%
\bigskip
\begin{center}
\begin{large}
\normalfont\itshape #1
\end{large}
\end{center}}
\renewcommand{\subsubsection}[1]{%
\vspace{2ex}
\noindent
\textit{#1.}---}
\renewcommand{\tableofcontents}{}

\setlength\parindent{0pt}

\begin{document}

\textbf{RE: PALA-11-19-4635-OA}\\
\bigskip
Dear Dr Benson,\\
\bigskip

We are very grateful to the three referees for their helpful and constructive comments, which we believe have helped significantly improve the manuscript.
Unfortunately, due to circumstances out of our control (pandemic, moving continent and new job), it took us far longer to reply to the reviewers requests than anticipated.
Fortunately, however, we have taken all of their comments on board, and respond to their points in detail below.

For improving clarity in this document, reviewer comments are displayed in blue, our responses in normal text and the changes to the manuscript in italic.
Additionally, we've uploaded two versions of the revised manuscript: both have the exact same content but one has all the changes and additions to the main text highlighted in yellow to help with reviewing.


\section{Editors' comments}

\textcolor{blue}{Both referees had much to say. My summary is that the manuscript is highly technical and difficult to follow as currently written. But I don't think it needs to be. This is important for publication in Palaeontology, and indeed in other journals. It is in all of our interests that the work can influence a wide audience. With this in mind, the choices of language and structure would benefit from some attention, specifically with a view to making it more comprehensible, and cleaning up some of the concepts so they are framed less ambiguously. I agree especially with the point made by referee 2 about the difference between (1) confounding covariation among character state distributions, and (2) informative covariation between character states that is induced by shared evolutionary history (the signal we're trying to infer during phylogenetic inference). This is not incidental. To many people this is the point of such studies.}

\textcolor{blue}{I have some illustrative comments of my own that help point to the type so issues I'm referring to. But note that these are not exhaustive. I'd like to see a deep and through consideration of these elements of communication, and those raised by the referees. It is likely that this constitutes fairly large modifications to the manuscript, but less so to the analyses and images.}

\textcolor{blue}{My example points:}

\begin{itemize}

\item{\textcolor{blue}{The abstract lists "shared phylogenetic history" as a factor that causes character correlation in a list of three factors. Approximately the same list of factors is presented in the introduction (p.5, bulletted), where "evolutionary dependence" is stated in place of "shared phylogenetic history". This seems confusing as the two lists seems to serve the same purpose but have non-overalpping content. Shared evolutionary history is a positive thing (we're trying to infer evolutionary history) whereas "evolutionary dependence" of traits seems like it could be negative in some circumstances (for the body elongation example, it may cause it to infer a clade of long-bodied taxa).}}

This is a very helpful point. We have adopted terminological consistency between the abstract and the introduction. We have now standardised the language to "evolutionary dependence" throughout the text and abstract.



\item{\textcolor{blue}{Matrices that were modified to minimise character differences are referred to as 'minimised'. However, in the context of the main questions, 'minimisation' of character differences corresponds to maximisation of character state correlations. Since these two concepts have opposite polarities, it would be easier for readers to follow the work if it consistently referred to one or the other - and I'd propose that referring to the amount of character correlation would be best and more straightforward for readers.}}

We have changed the naming of the matrices and the trees throughout the whole manuscript to now only focus on the character correlation aspect (\textit{c.f.} the character difference). The \textit{Normal} matrix/tree is now the \textit{Unperturbed} matrix/tree the \textit{Maximised (difference)} matrix/tree is now the \textit{Minimised (correlation)} matrix/tree and the \textit{Minimised (difference)} matrix/tree is now the \textit{Maximised (correlation)} matrix. The \textit{Randomised} matrix/tree stayed unchanged.



\item{\textcolor{blue}{'Normal' has multiple meanings and mathematically could mean 'orthogonal'. In the way the authors use it, it comes across as colloquial, and it doesn't contain much information (there is no objective meaning of 'normal' in this context). A more intuitive term might be 'unmodified' or 'simnulated', or something else, there are probably better options.}}

We have now changed mentions to the \textit{Normal} tree as "the topology inferred from the unperturbed matrix" to avoid any confusion throughout the manuscript.



\item{\textcolor{blue}{'In a few places 'phylogenetic signal' is used to refer to a situation where a topology is well-supported. But I had the impression this might not strictly correspond to support of the "true" phylogeny (correct me if I'm wrong however). Phylogenetic signal is a widely-used phrase with several different potential meanings. So to avoid confusion it would be useful each time if the text referred explictily to support for the "true" phylogeny underlying the simulations, or to support for nodes in the inferred phylogeny, whether they are present in the true phylogeny or not. Or tended to produce strong support for incorrect topologies.}}

We have now replaced mentions of "phylogenetic signal" to "\textit{phylogenetic information}" throughout the whole manuscript to avoid any confusion with other uses of the term "phylogenetic signal". Here we mean information (code) used to infer phylogenies.

\end{itemize}












\section{Referee 1}

\subsection{General notes}
\begin{itemize}
\item{\textcolor{blue}{Please supply your main text in Word format rather than PDF; please supply all figures as separate files and include a list of figure captions at the end of the manuscript rather than embedded in the text. Please check journal submission guidelines:}}

This manuscript was written in \LaTeX\ and no proprietary softwares (e.g. Word) where used. We have added an .odt format version of the manuscript as well as all the figures as separated files with the figure captions at the end of the manuscript.

\item{\textcolor{blue}{Please respond directly to all referee comments, including the technical comments below. It is particularly important that you explain your reasoning if you have not followed any of the suggestions made in the reports.}}

\item{\textcolor{blue}{Please upload your response to the reviewers as a separate document designated as a 'Supplementary File' with the other submission files. This will pull it into the automatically generated proof that is available to all reviewers.}}

\item{\textcolor{blue}{Do any of the authors have an ORCID iD? This is a digital identifier that allows all publications to be permanently linked to the individual authors, even if they change institution or name. It is free to register, and we strongly recommend that at least the corresponding author should do so. There is more information available here: http://orcid.org/. If each author can link their ScholarOne account directly to ORCiD (which requires logging into ORCiD to authorise) they will automatically be added on this and any future paper you might submit, and metadata for a published paper will automatically be sent to their ORCiD account. I have attached some guidelines on how to do this. You can also simply add IDs to the manuscript (with the affiliation information) but our publisher cannot push publication data to ORCiD without consent.}}

We have now added our ORCiDs to the author list.

\end{itemize}

\subsection{Data}

\begin{itemize}
\item{\textcolor{blue}{Thank you for uploading your data to GitHub. Please replace the link in the text with a citation to a data reference: Guillereme, T., Pastore, A. and Brazeau, M. 20xx. Influence of different modes of morphological character correlation on phylogenetic tree inference. GitHub. https://github.com/TGuillerme/CharactersCorrelation}}

We have now added a proper reference for the data and code.

\item{\textcolor{blue}{We no longer upload any supplementary information on our publisher's website. This information, including figures, needs to be properly curated. Can your supplementary figures be uploaded to GitHub or Figshare? If not, I suggest Dryad (the Palaeontological Association will pay the fee for a Dryad record for any data package associated with a paper in one of its journals). Please contact me directly for an upload link if you wish to use Dryad (other repositories such as MorphoBank are also acceptable). Any text should be supplied in a non-proprietary format (txt files can be supplied alongside a styled pdf version if you prefer). Dryad encourages uploading data in multiple formats if styling is important.}}

We have now added all the supplementary documents as a FigShare item with it's own DOI.

\item{\textcolor{blue}{As above, any data citations in the text should be linked to a data reference including the doi for the record.}}
\end{itemize}

\subsection{References}
\begin{itemize}
\item{\textcolor{blue}{Where you have two references with identical citations (e.g. Guillerme \& Cooper 2016) please list these in the order in which they are cited in the text, irrespective of publication order, or alphabetical order of second author. Therefore Guillerme \& Cooper 2016a should be cited first, whichever reference this is. Having said that, I only found references of Guillerme \& Cooper 2016b so please check this.}}
\end{itemize}

We have now fixed this issue.

\subsection{Figures}

TODO: @@@ I'll do that later

\begin{itemize}
\item{\textcolor{blue}{Please note that although figures may be reproduced in colour in the online edition, there is a fee to reproduce them in print. If a figure appears in colour online and greyscale in print, the same file will be used for both, so please carefully consider your use of colours to make sure that it works in shades of grey.}}
\item{\textcolor{blue}{Please supply all of your figures at a resolution of 600 dpi; preferably in tif format using LZW compression. Embedded photographs are fine at 300 dpi, but if any labelling is included the overall figure will require 600 dpi for printing. Please do not use jpg compression at any stage. Final widths should be either single column (80 mm), 2/3 page width (110 mm) or double column (166 mm). Please see the attached figure
guidelines.}}
\item{\textcolor{blue}{Please confirm the final intended width of each figure (166 mm = full page; 110 mm = 2/3 page; or 80 mm = single column) by adding this to the file name (add 166, 110 or 80).}}
\item{\textcolor{blue}{Fig. 1: Please note that although the journal uses British English, it uses suffix variants - ize, -iza and -izi where appropriate (rather than -ise etc.) This would be changed in the main text but it would be helpful if you could correct any instances on figures (e.g. Randomized, Maximized, Minimized). Similarly, journal style uses single quotes throughout (e.g. 'True' tree)}}
\item{\textcolor{blue}{Fig. 2: Note spelling as above. Please do not refer to colours in the caption if this is intended for printing in colour. Generally, a visual legend is much better on figures anyway (there is room for this in the top right hand corner; it does not require a title).}}
\item{\textcolor{blue}{Figs 3, 4: These figures would require colour both online and in print to be clear. If you do not have funding available to pay the colour printing fee (£150 for one figure, plus £50 for each additional one) please state this clearly in your cover letter as the Palaeontological Association is willing to cover the cost of essential colour where alternative funding is not available (at the Editor-in-Chief's discretion). Please view these two figures at full page width (166 mm wide) and check that all text labels are clearly visible. The axis labels in particular are currently too small; generally, text sizes should be in a range equivalent to 6–10 pt Arial (viewed at 100\%).}}
\end{itemize}

\subsection{Tables}

TODO: @@@ I'll do that later

\begin{itemize}
\item{\textcolor{blue}{Please supply tables either separately, or at the end of the manuscript.}}
\item{\textcolor{blue}{Please note that captions would be altered to include a single, short descriptive phrase, with all additional information/abbreviations moved to a footnote.}}
\end{itemize}















\section{Referee 2}

\begin{itemize}

\item{\textcolor{blue}{Lines 22-23: 'probabilistic methods can easily accommodate for a random correlation between characters.'
\\
'random correlation' sounds like an oxymoron! I think that what the authors really mean is moderate amounts of correlation that basically mimic accelerated local rates of change on a small proportion of branches. From what the authors state later, it's not the randomness that is important, but that it occurs infrequently relative to total taxon sampling.}}

We have now changed "random correlation" to "\textit{randomised} correlation" throughout the manuscript to better reflect this comment.

\item{\textcolor{blue}{Lines 50-55: Types of dependence.
\\
I found the definitions here adequate, but I think that they should be tied more closely to aspects of correlated character change as discussed elsewhere in systematics literature and even general evolutionary biology. I, myself, tend to think of correlated characters as falling into two general categories: functional suites and integrated modules. (Note that these need not be exclusive; modules can link functionally linked characters.)
\\
The reason why I prefer this is that we can model both functional suites and integration with extended Mk models. Suppose we have two binary characters. We will use matrix exponentiation to describe the probabilities of transitions between character combinations. Under independent change, our model would look like:
\\
Q=...
\\
We'll set a=1 and let's say we have a rate of change of $\pi$=0.05. Our transition matrix for time t=1 is now mexp($\pi$Qt) =
\\
Note that if we calculated the probability of going from 0 to 1 for either character, then we've have almost the same value: the difference is that there is some small chance that, over the same amount of time, we also went from 0 to 1 for the other character, and thus from 00 to 11. The probability of stasis is the probability of stasis for one character squared.
\\
Change here is independent; the condition of Character 1 has no effect how Character 2 evolves, and the two characters do not change together. (This is assuming continuous time change.) We can introduce a new Q matrix that allows Characters 1 and 2 to affect each other, and in which there are joint changes altering both characters:
\\
Q=...
\\
The two d values reflect what I think that authors mean by 'evolutionary dependence': 11 is an 'optimal' condition, and once you get to 10 or 01, then the probability that the other character changes increases (d1). Conversely, once you get to 11, then reversing back to 01 or 10 is less probable (d2). This is the sort of change we would expect in mosaic functional evolution, in which new changes were 'improving' on the overall morphotype.
\\
The off-diagonal reflects (j) joint change: a change 'deeper' in the organisms that affects both characters. Integration predicts this: we alter a module of linked characters, and we change states on multiple characters. We can get some independent change because different parts of the module can (sometimes!) change on their own; however, there are many other ways to change the module that will affect both (and possibly other) characters. Heterochronic shifts are one particular version of this model.
\\
This is where I was a little uncertain: does integration fit in with intra-organismal dependence? If so, then the authors should state that. At any rate, given the ubiquity of papers describing integration and modularity papers, I think that it would help the readers to understand the correlation models if the discussion was presented in terms of integration/modules and functional suites.}}


We agree with these two broad categories of correlation types, but we didn’t find them adequate to characterise the range of possible correlations that could be encountered.
Given the nature of our work, we thought ‘atomizing’ the types of correlations possible would be useful in preventing misinterpretations of our results.
Nevertheless, we think the reviewer has raised a really important point that we do not wish to dismiss.
We have modified our discussion of the types of correlation and made specific reference to the two types mentioned by the referee.

\item{\textcolor{blue}{Lines 58-59: 'For example, two characters 'tail colour' and 'tail length' could be coded two times as an absence for a taxon with no tail (Wilkinson, 1995b; Brazeau et al., 2018).'
\\
Coding this as inapplicable makes this much less of a problem for likelihood, where it is treated as an unknown. In likelihood, unlike parsimony, the ancestral condition is not fixed: so, if tails are found in two subclades, then the marginals for the 'tailless' paraclade will have low values for tail being present: but there is some small probability that the tail was there, and then lost in parallel. If that tail was there, then it had States A, B, C, etc., for length, color, etc. Because we are multiplying this, it's really P[tail present] X P([tail state A]+tail state B)) etc.}}

We have added this precision in the text:

\textit{For example, two characters ``tail colour'' and ``tail length'' could be coded two times as an absence for a taxon with no tail (Wilkinson, 1995b; Brazeau et al., 2018). Note that there are a lot of alternative methodological choices for this example (e.g. using inapplicable tokens; Brazeau 2011) and that these choices affect results more or less depending on the estimation method (parsimony methods contrast with probabilistic methods were ancestral conditions are respectively fixed or not). Line @@@.}

\item{\textcolor{blue}{Lines 69-71: 'Evolutionary dependence itself requires the accurate resolution of a phylogenetic tree, and is best determined by independent character sets.'
\\
I would add three points here. One, with fossil data (such as readers of Palaeontology use), we have very few options for independent character sets. Two, as the authors cite above, we have reason to think that correlated change happens with molecular evolution. The really important point is that what we want are character data that we know had independent character change: but no biological data are above suspicion here. Three, we do not need accurate resolution of trees if we have sets of characters that we suspect are correlated. Given such a set, we can run two sets of Markov Chain Monte Carlo analyses: one in which characters in the set evolve independently, and one in which we use more complex models in which the states are not the states of one character, but the combinations of states from 2+ characters (Billet \& Bardin 2018 Syst. Biol. 68:267, Pagel 1994 Proc. R. Soc. B 255:37). We then can assess whether the correlated change model produces appreciably better posterior probabilities in the same way that we can contrast any other pair of different models.
\\
What Guillerme et al. are assessing is a little bit different from what Pagel and later workers considered: what happens if there is correlated character evolution happening among unknown sets of characters? Billet \& Bardin's study focused on mammal teeth: but even non-biologists would expect mammal teeth to show correlated character change. This raises the issue with which readers of Palaeontology will be familiar: how do I have reason to suspect that there is correlated change in some group of characters, and what happens if I fail to guess? What this simulation study offers is an idea of what happens when there is some amount of correlated change, but we just do not know how much.}}

We are grateful for this insight and have added it to the manuscript:

\textit{Evolutionary dependence itself requires the accurate resolution of a phylogenetic tree (but see useful methods from Pagel 1994 and Billet and Bardin 2019), and is best determined by independent character sets. The later being commonly hard to get because of the nature of the data: some common type of data (e.g. fossil data) offer very few options of independent character sets and no biological datasets can really be independent. Line @@@.}

\item{\textcolor{blue}{Lines 123 – 124: In order to measure this, we used the following scaled Hamming distance between two characters:
\\
I think that it would be worth comparing this to pairwise dissimilarity among taxa. After all, there have been numerous papers published in Palaeontology that use pairwise dissimilarity (also called phenetic dissimilarity or phenetic distance) to measure disparity. The difference is basically R-mode vs. Q-mode: for phenetic dissimilarity, we count the number of differences between two taxa and divide by the total number of comparable characters to get the proportion of different characters. Here, we count the number of differences between two characters and divide by n-1 to get (almost) the proportion of differing taxa. It basically is like generating scatterplots for every pair of characters, and then counting the number of taxa that fall above or below the line bisecting the axes (0,0 : m,m).}}

In this sense, this metric is used to compare \textit{characters} with each other to create matrices with more or less induced correlation. Our character correlation metric (character difference) is used as an optimisable variable to increase or decrease correlation in our simulated matrices. We believe that the reviewer here suggests comparing character distances (phenetic distance) between \textit{taxa} (cf. between characters). We are not sure what this would bring to the present manuscript since this was not the focal point of this study. In fact, one could argue that this phenetic distance is somehow recovered when inferring the topologies based on the modified and un-modified matrices. But again, this is not what we where trying to explore here.




\item{\textcolor{blue}{Lines 131-132: 'We standardised each character by arbitrarily modifying their character state tokens (or symbols) by order of appearance.'
\\
I think that the authors should state 'by order of derivation' or 'order of evolution.' I realize that their simulations evolve up to n taxa and then attempt to reconstruct phylogeny. However, many simulations in palaeobiological studies sample simulated taxa over time. In such cases, the order in which states evolve and the order in which they are sampled might not be identical, particularly for characters that evolve rapidly relative to sampling rates.}}

We apologise for the confusion here: by ``order of appearance'' we meant:
\textit{order of appearance in the matrix - for example for the first character (1st column) in the matrix, if the state coded for the first OTU (1st row) was \texttt{3}, it was translated into \texttt{1} for the whole character (across the 1st column); if the state coded for the second OTU (2nd row) was \texttt{0}, it was translated into \texttt{2} for the whole character, etc...} and have now specified that in the manuscript on line @@@.



\item{\textcolor{blue}{Lines 150 – 152: 'The 'morphological HKY-binary' model (O'Reilly et al.,
2016) which is an HKY model (Hasegawa et al., 1985) with a random states frequency (sampled from a Dirichlet distribution Dir(1, 1, 1, 1))...'
\\
The authors should state that this allows variation in gains/losses of present absent characters to vary without specifying which characters are prone to gain/loss. The authors also should note that because it's (1,1,1,1), the expectation is that gains and losses will be equiprobable. This does raise the question: why didn't they use a beta distribution (1,1) given that they were dealing with 2-state characters? That's just the special case of the Dirichlet. Using a 4 state character like this seems a little weird because net stasis will include both net stasis and net transition, and thus is P[net stasis] + P[net transition]; net change now is twice net transversion and thus 2*P[net transition]. So, this will not quite be the same distribution as beta(1,1); the expectation will still be 50:50, but (I think!) the variance on the state-acquisition bias will be a little greater under the 'lumped' 4-state Dirichlet than under the Beta.}}

We used the Dir(1,1,1,1) distribution to be consistent with previously published simulations (e.g. O'Reilly et al. 2016).
We have however specified the following:

\textit{This allows the changes in character states to vary without specifying which change is more likely, hence expecting an equiprobable change in character state in all directions.} Line @@@.



\item{\textcolor{blue}{Lines 155-156: 'To generate more than binary states characters, we used the Mk model (Lewis, 2001).'
\\
How did the Mk model generate the 3-state characters? Shouldn't the authors have used a Dirichlet (1,1,1) (or a Dirichlet (1,1,1,1,1,1) with lumping of 1\&2, 3\&4 and 5\&6) to actually generate the characters with individual rate bias?}}

We generated multi ($>2$) state characters with the Mk model using the \texttt{ape::rTraitDisc} function (through \texttt{dispRity::sim.morpho}) that calculates the transition probabilities for each branch using any sized Q matrices. We specified this in the text:

\textit{we used the Mk model (Lewis 2001). We drew the number of character states with a probability of 0.85 for binary characters (using a 2x2 transition matrix Q) and 0.15 for three state characters (using a 3x3 Q matrix)}. Lines @@@



\item{\textcolor{blue}{Lines 159-161: 'For each character, both models ('morphological HKYbinary' or Mk) were chosen randomly and run with an overall evolutionary rate drawn from a gamma distribution ($\beta$ = 100 and $\alpha$ = 5).'
\\
The authors should make a couple of points clear here. The first is that they should make clear what this section is doing vs. the prior section. I would state plainly that the prior section was about generating variation in state gain/loss biases. This part is about generating variation in character change; basically, you are allowing each character to change at its own rate and (when it does change) to have its own bias in gains/losses of states; however, in all cases, every character's particular rate and gain/loss bias is drawn from the same distribution.
\\
The second is that this is different from the way systematists usually use gamma distributions for rate variation, in which they have a separate rate parameter $\pi$ (I will use $\pi$ rather than $\alpha$ as Lewis did, simply to avoid confusion between rate parameter and the Gamma shape parameter $\alpha$). They then set $\beta$=$\alpha$ so that the mean of the Gamma, $\alpha$/($\beta$=$\alpha$), is 1.0 and the variance, $\alpha$/($\beta$=$\alpha$)2, increases as $\alpha$ decreases. The expected rate of change then is $\pi$ and the variance is $\pi$/$\alpha$.
\\
By using separate shape ($\alpha$) and rate ($\beta$) parameters without any independent rate parameter, these authors are making a mean rate $\pi$ = $\alpha$/$\beta$ = 0.05 with variance of the rate being 0.0005. I think that the authors should just state this. (Also, I'm not sure that everyone knows reflexively what the mean and variance are given $\alpha$ and $\beta$, so it's probably best to remind them.)}}

We thank the reviewer for this clear explanation and agree that it is always best to be extra clear and remind the reader what these values actually mean. We've added the following precisions:

\textit{This simulated, for each character, a variation in the change of character state at different rates while ensuring that these rates were drawn from the same distribution, resulting in an overall mean character rate of 0.05 with a variance of 0.0005.} Line @@@



\item{\textcolor{blue}{Lines 161 – 163: 'low evolutionary rate values allowed reduction in the number of homoplasic character changes, thus reinforcing the phylogenetic information in the matrices'
\\
Again, you need to make sure that the readers know how to convert Gamma parameters to rates! Also, is a rate of 0.05 really all that low? That's a pretty typical rate per million years for fossil invertebrates.}}

We explicitly specified the overall mean and variance of the rates distributions (see above) and removed the term ``low'' to reflect this comment.



\item{\textcolor{blue}{Lines 163-164: 'We re-simulated every invariant characters to obtain a matrix with no invariant characters in order to better approximate real morphological data matrices.'
\\
Ah, but that also means that the true rate is >.05! Programs like RevBayes take into account ascertainment bias (unscored invariant characters), and individual runs could be limited to examine only variant characters; wouldn't doing the latter and then accommodating ascertainment bias in the analyses be a little more realistic? (This is probably not a major issue, so I would not worry over much about it; however, it is worth mentioning ascertainment bias at this point and why that makes what you are doing more realistic than keeping invariant characters in the simulations.)}}

We now specify all this as follows:
\textit{We re-simulated every invariant character to obtain a matrix with no invariant characters (making the effective mean rate to > 0.05). Although invariant characters are important for effectively measuring the true evolutionary rates, many empirical matrices do not contain invariant characters which is in turn taken into account by software that accommodate for this ascertainment bias.} Line @@@



\item{\textcolor{blue}{Lines 165-167: 'we only selected matrices with Consistency Indices (CI) superior to 0.26 (Sanderson and Donoghue, 1989; O'Reilly et al., 2016).'
\\
I have discovered that younger workers often do not know what the Consistency Index is! So, you might explain that this means about 3.85 (1/0.26) changes per derived character; if we have 50 binary characters, then it would be 192 changes (50/192 = 0.26).}}

We've now added the following precision:

\textit{To ensure that our simulations were reflecting realistic observed parameters, we only selected matrices with Consistency Indices (CI) superior to 0.26 (expecting at least 3.85 (1/0.26) changes per derived character; Sanderson \& Donoghue 1989; O'Reilly et al. 2016).} Line @@@



\item{\textcolor{blue}{Lines 200 – 202: 'Bayesian inference was run using an Mk model with ascertainment bias and four discrete gamma rate categories (Mkv 4G) with an variable rate prior an exponential (0.5) shape'
\\
I think that much more explanation of what these parameters and terms mean is in order! I note above that the authors should discuss ascertainment bias and why their simulations made sure that all characters had at least one derived taxa. At the very least, ascertainment bias needs to be defined here: I have found that a lot of morphological systematists do not know what it is.
\\
The authors use four Gamma rate categories: but how are the characters categorized? Again, the authors should remind the readers that this means that we are assuming that characters have a 1 in 4 chance (or however you divide the Gamma) of having the 1st, 2nd, 3rd or 4th quartile rate without assuming that any one character is fast or slow.
\\
Also, it should be explained that the Gamma distribution itself is varied over the runs, so that the tree will be considering new topologies, new average rates and new rate distributions. What this means is that the shift in the $\alpha$=$\beta$ parameter is drawn from an exponential distribution with a mean of 0.5. However, I think that it is also worth reminding the reader that every generation in the MCMC analysis will consider altered topologies, average rates and Gamma shape parameters; improvements are kept and (sometimes!) worse propositions are accepted, too, in order to better search hypothesis space.}}

We thank the reviewer for these precisions and again agree that it is important to make sure the reader understands the intrications of these parameters:

\textit{Bayesian inference was run using an Mk model with ascertainment bias and four discrete gamma rate categories (Mkv 4$\Gamma$) with an variable rate prior an exponential ($\alpha$=0.5) shape. We've used the Mk model with ascertainment bias (Mkv) to correct for the fact that no invariant characters were present in the matrix (as is often the case with morphological data) allowing the model to estimate the correct evolutionary rates (see above). We used four discrete character rates categories making the assumption that each character has an independent 1/4 chance to be in the 1st, 2nd, 3rd or 4th quartile of the rate distribution. Finally we allowed that the average rate of this distribution (alpha) is drawn from an exponential distribution (with a mean of 0.5) in every iteration of the MCMC allowing an efficient exploration of the parameter space.} Lines @@@



\item{\textcolor{blue}{Lines 202-204: The MCMC was ran over two runs of 6 chains each for a maximum of 1 x 109 generations with a sampling every 200 generations with an automatic stop if the average standard deviation of split frequency (ASDSF) fell below 0.01.
\\
The authors need to state that getting the ASDSF down to <0.01 is taken to mean 'convergence': that is, the individual Markov Chain has not been finding any major improvements in a long time, and that it is has been sampling the same (and 'good') part of hypothesis space for a while.
\\
To this end, the authors also have to define what convergence is! The next paragraph discusses what they do when they fail to reach convergence, but readers who do now know what convergence is (in this context) are going to be baffled.}}

We've added the following to explain the convergence and the ASDSF more clearly:

\textit{We used this method to diagnose when the 12 chains reached a similar region in the parameter space. In other words, we considered that when the differences in solutions between the chains reaches a stabilised solution independently (ASDSF < 0.01), then the MCMC converged on the “correct” inference} Lines @@@



\item{\textcolor{blue}{Lines 215-218: 'We measured both the Robinson-Foulds distance (Robinson and Foulds, 1981) and the triplets distance (Dobson, 1975) between the trees inferred from the 'maximised', 'minimised' and 'randomised' matrices and the tree inferred from the 'normal' matrix.
\\
I think that a brief explanation of the metrics is in order here. If nothing else, then move lines 230 – 231 up to this paragraph so that readers will know right away why you are using both metrics.}}

We now briefly define the two metrics when mentioning them:

\textit{we measured both the Robinson-Foulds distance (the differences in clade compositions; Robinson \& Foulds 1981) and the triplets distance (the differences in tips positions; Dobson 1975) between the trees inferred}. Lines @@@

\end{itemize}
















\section{Referee 3}

\textcolor{blue}{Sadly however, I am skeptical that this paper really addresses the central issue with character correlation and inferring phylogenies. Particularly, their main conclusion is that character correlation is a 'good' thing, i.e. having increased character correlation actually results in more accurate topologies.} 

This isn't quite a correct interpretation of the paper. We appreciate therefore that our explanation may not have been entirely clear and must have caused some confusion. We think, however, it would be an unreasonable standard to expect one paper to solve a monumental problem such as character correlation effects on phylogeny. We don't propose to solve problem in its totality. We have now revised the introduction and discussion to clarify the message of the paper; if character correlation is highly stochastic within a dataset, the individual correlations effectively cancel each other out. Whereas, if correlation is intentionally decreased by the investigator, this can seriously bias the results. Similarly, if it is maximized by the investigator, as expected, it will also bias the result.


\textcolor{blue}{One of the central conclusions (from the abstract) is 'This means that datasets with low correlation between characters will make it more complicated to estimate a correct topology'. This is not a novel conclusion, nor does it really have anything to do with the issue that the authors claim to address. All this is really confirming is that inferring phylogenies from matrices that contain limited phylogenetic signal is difficult; it is not related to how correlated characters that show homoplasy impact our ability to infer accurate phylogenetic hypotheses. If character correlation really were a positive influence on topological accuracy then character matrices should be coded using a/p coding (rather than using hierarchical characters with some taxa being inapplicable for certain characters) as this coding method would maximize character correlation (e.g. Pleijel 1995, Cladistics).}


We do agree that these are valid limitations from this study and have acknowledged them more clearly:

\textit{Third, by not focusing on modeling the sources of character correlation specifically (see section “Distinction between different character correlations” above), we do not allow to distinguish the specific effect of each correlation for downstream analyses (see Sansom et al. 2017 for such an application on dental morphological data). However, these sources are often not available to authors, either because of the data availability or the resources available to the workers (i.e. time and money) and thus we here propose an approach showing the effect of character correlation when the sources of correlation are effectively unknown.} Lines @@@



\textcolor{blue}{Character correlation should only be a problem for inferring an accurate topology if the correlated characters contain misleading phylogenetic signal and change multiple times in concert on the tree, i.e. show coordinated homoplasy.  This is effectively the 'Intra-organismal dependence' type of character correlation that the authors identify in the introduction, the type of character correlation that is probably of most concern regarding inferring phylogenetic trees. This type of dependent character state change is not only an issue for morphological phylogenetics, but is likely also a problem for molecular data. For example the presence of particular amino acids at particular sites in proteins is fundamental for determining 3D protein structure, limiting the substitutions that can occur at particular sites if proteins are to maintain this 3D structure, which is similar to the tooth occlusion example that is cited in the text. This a particularly relevant and interesting example, especially given that an empirical analysis investigating the performance of dental data from mammals found that they recover phylogenies that are more in conflict with those inferred from molecular data (Sansom et al. 2017) citing character non-independence as a possible factor driving this incongruence. This paper is absent from the bibliography and should certainly be discussed, especially as the conclusions of this paper differ so radically.}

We thank the reviewer for drawing our attention to this paper (Sansom et al. 2017) and the DNA example and have now added that to our manuscript:

\textit{It is also worth noting that the evolutionary and intra-organismal dependences induced correlations are by no means restricted to the use of discrete morphological data. In fact, by essence, these correlations appear to any type of data whether continuous (e.g. linear measurements or morphometric geometric measurements) or non-morphological (e.g. behaviour, or - much more commonly used - molecular data, Sansom et al. 2017.} Lines @@@




\textcolor{blue}{In the discussion (P6. Line 339) the authors state '(1) evolutionary correlation is implied by simulating the characters using Birth-Death trees; and (2) intra-organismal correlation is present in the matrices for the characters randomly simulated but sharing similar evolutionary simulation regimes.  However, the effect of these sources of correlation was out of the scope of this study and would have required a posteriori changes to the matrices'. This means that characters that are correlated due to shared evolutionary history and characters that show coordinated homoplasy are not distinguished by downstream analyses, limiting the applicability of the results for understanding issues raised by real world examples (such as the dental data analysis by Sansom et al. 2017, Sys Bio).}

We do agree that these are valid limitations from this study and have acknowledged them more clearly on lines @@@ as highlighted above.

% This is the text it should refer to: \textit{Third, by not focusing on modeling the sources of character correlation specifically (see section “Distinction between different character correlations” above), we do not allow to distinguish the specific effect of each correlation for downstream analyses (see Sansom et al. 2017 for such an application on dental morphological data). However, these sources are often not available to authors, either because of the data availability or the resources available to the workers (i.e. time and money) and thus we here propose an approach showing the effect of character correlation when the sources of correlation are effectively unknown.} Lines @@@


\textcolor{blue}{In the conclusions the authors state: 'However, in empirical datasets, if character difference is not driven by selection (e.g. pleiotropy) correlation is likely be cancelled out if the correlated characters are randomly distributed with respect to traits.' I am not entirely sure what the authors mean by this sentence. I think that what they are suggesting is that character correlations that arise due to selection will have different impacts on accuracy of phylogenetic reconstructions vs those that arise from drift, which really is what this paper should be trying to address. Instead the analyses pool these types of character correlation rather than trying to disentangle them.}

We have removed this sentence altogether since it was not impactful for the last part of the conclusion. We agree though that it would have been interested to look at how correlation arise between traits in organisms (e.g. through drift or/and selection) and how that would translate into discrete morphological characters but that was outside of the scope of this project.


\subsection{Simulation protocol and methodology}

\textcolor{blue}{I am not sure that I understand the logic of only comparing the results of the phylogenetic analyses to what the authors call the 'normal' tree and not the tree that generated the data. Surely the whole point of this exercise is to understand if character correlation actually results in misleading phylogenetic inferences.}

The purpose of the “normal tree” (now described as the ``topology inferred from the unperturbed matrix'') is to isolate the variables being studied. Indeed, if the whole point of the exercise is to understand if character correlation actually results in misleading signal, one must isolate that variable from all others. This difference is crucial because our methods did not directly simulate covariation. We simulated datasets and then perturbed them. Thus the experimental ‘true tree’ (now described as "the starting tree") is the one recovered by the analyses in the absence of any of our experimental perturbations. Comparison with the simulation (i.e. “starting tree”) would introduce additional stochastic variables that arose during the unperturbed simulation of the datasets. Comparisons with that tree would be unable to isolate the effects of our experimental perturbations.



\textcolor{blue}{It strikes me that the way in which they treat their morphological matrices just results in the 'minimised' matrix having the phylogenetic signal that supports the 'normal' tree reinforced as the correlated characters will essentially just contain the same phylogenetic signal that supported the 'normal' tree, though not necessarily the true tree.  For many of the simulated matrices, the normal tree could share very few clades with the tree that was used to generate the data itself, which is clearly demonstrated in O'Reilly et al 2016 (where the same binary HKY simulation method was also used). In this paper, the accuracy of the trees from downstream phylogenetic analyses are often very different to the generating tree. In any revision, I would want to see the impact of comparing the simulated matrices to the generating tree.}

We’ve added the comparisons between the trees from the modified matrices and the "starting tree" in the supplementary materials.
However, as stated above, what these results actually show is the effect of A: changing the character correlation + B: the effect of phylogenetic inference + C: the effect of simulating discrete morphological characters. This last component C is not the focus of our study (as specified on lines @@@) and thus is removed from the main manuscript. In fact, there is a really rich amount of papers trying to understand the effect of B and C with no consensus so far (i.e. simulating realistic evolutionary data is complicated AND inferring a realistic evolutionary history from real data is complicated). Therefore, we chose to only focus on A and B and correct for C by effectively comparing to the topology inferred from the unperturbed matrix rather than the starting tree (i.e. making the results relative to the ``best'' inference possible with the simulated data). 



\subsection{Minor/stylistic comments}

\textcolor{blue}{The merged PDF for review has identical font size for figure/table captions and the main text, making it difficult to read fluently.}

We have changed this in this revised version.

\textcolor{blue}{The text has many typos, mostly pluralization (e.g. character vs characters). Randomised is misspelled in Fig 1. Please check the text and figures thoroughly.}

We have fixed the typo in figure 1 and checked the rest of the manuscript more thoroughly.

\bibliographystyle{sysbio}
\bibliography{References}



\end{document}
